package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "website-my",
-  "version": "0.0.0",
+  "version": "1.1.0",
```

Comment:
Having a small doubt, the previous version was 1.0.0 right ?? but here it was 0.0.0

Reply:
It was never made v1.0.0 in the package json 😛 
npm by default gives us v0.0.0

Reply:
Got it approving the PR